local-cli/rnpm/link/test/android/patches/0.17/makeImportPatch.js

```diff
@@ -7,7 +7,7 @@ const makeImportPatch = require('../../../../src/android/patches/0.17/makeImport
 const applyPatch = require('../../../../src/android/patches/applyPatch');
 
 const projectConfig = {
-  mainActivityPath: 'MainActivity.java',
+  mainFilePath: 'MainActivity.java',
```

Comment:
To be honest it should be `MainApplication.java` but I assume we are not planning to update tests in this PR?